### DISCUSSION TITLE
Isaac Lab Office Hours

### DISCUSSION THREAD

**Discussion by rthaker01** [isaac-sim/IsaacLab#1438] [Announcements]
Join NVIDIA robotics engineers in the upcoming office hours to learn more about robot learning and robot policy training using NVIDIA Isaac Lab.
Save the date for the livestream happening **Nov 20th at 9 am PDT**
Link: https://www.addevent.com/event/Uz23738360

**Comment by RandomOakForest** [comment]
Recording: [Deep-Dive Into Isaac Lab Workflows | Isaac Lab Office Hours](https://www.youtube.com/live/_UHxP0FbOws?feature=shared)
